config/routes.rb

```diff
@@ -1,21 +1,36 @@
 Rails.application.routes.draw do
 
-  root 'display_tasks#display_tasks'
+  root 'display_tasks#index'
 
   # route to landing page that displays all tasks
-  get '/tasks' => 'display_tasks#display_tasks'
-
-  # route to page with single task
-  get '/task/:task_id' => 'display_tasks#display_task'
+  get '/tasks' => 'display_tasks#index'
 
   # route to page with form for entering new task
   get '/tasks/new' => 'display_tasks#new'
 
+  # route to page with single task
+  get '/tasks/:task_id' => 'display_tasks#show'
+
+
   # new task gets posted here
   post '/tasks' => 'display_tasks#create'
 
-  delete '/task/:task_id' => 'display_tasks#destroy'
+  # goes to the edit page with task info filled in
+  get '/tasks/:task_id/edit' => 'display_tasks#edit'
+
+  # updates single task
+  patch '/tasks/:task_id' => 'display_tasks#update'
+
+  post '/tasks/:task_id' => 'display_tasks#mark_complete'
+
+  # delete single task
+  delete '/tasks/:task_id' => 'display_tasks#destroy'
+
+  get '/people' => 'people#index'
+
+  get '/people/:person_id' => 'people#show'
 
+  get '/people/:person_id/tasks' => 'people#tasks_for_person'
 
 
```

Comment:
Your routes look good. I think you can remove the model-specific ID names and favor the basic `id` since this is an accepted standard. 
The edit route seemed to be a tough one for folks and yours looks great!